### DISCUSSION TITLE
Possibility to set the notification / alerts

### DISCUSSION THREAD

**Discussion by ggiak** [add2cal/add-to-calendar-button#510] [Ideas / Requests]
Would it be possible somehow to set the notifications as well?

![image](https://github.com/add2cal/add-to-calendar-button/assets/20743694/4b89bbb8-c55b-46d0-8113-c222008a8e33)

**Comment by jekuer** [comment]
Hi @ggiak ,

I really understand the need here, but unfortunately, we do not have this topic on our roadmap.
Basically, because it is impossible to have this as a stable feature.
Some calendars offer an option to provide this information, but under very limited conditions. And it then event gets overwritten by the user's settings. The calendar providers do this for a reason - it is simply bad UX and potentially harmful, if an external tool can push notifications via the calendar app.
Bottom line: If we would offer this as a feature, it would only lead to a lot of frustration, since it would not work 90% of the time.

All the best!
Jens


_Similar questions and discussions:_

- https://github.com/add2cal/add-to-calendar-button/discussions/348
- https://github.com/add2cal/add-to-calendar-button/issues/26
